phoenix-core/pom.xml

```diff
@@ -523,6 +523,13 @@
       <artifactId>hamcrest-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why is this needed ?
Ideally Phoenix-core doesn't have anything to do with sqlline.